environment/stats/src/cloud/benchflow/collectors/stats.go

```diff
@@ -20,11 +20,12 @@ import (
 type Container struct {
 	ID           string
 	statsChannel chan *docker.Stats
+	doneChannel chan bool
```

Comment:
@VincenzoFerme Because golang panics when you try closing a channel that;s already closed, and you can;t tell if it's closed until you attempt reading from it, which causes problems. This makes it work for multiple containers being collected from.